### PR TITLE
feat(GUI): add drive quantity to flash analytics

### PR DESCRIPTION
**Pull request by Shou**
We add a field `driveAmount` to the flash analytics events in the image
writer.

Change-Type: patch

### CODE REVIEW

File: lib/gui/app/modules/image-writer.js [hunk 1]
```diff
@@ -255,6 +255,7 @@ exports.flash = (image, drives) => {
   const analyticsData = {
     image,
     drives,
+    driveAmount: drives.length,
```
Comment (jhermsmeier) [inline lib/gui/app/modules/image-writer.js]: How about `driveCount` here? `driveAmount` sounds a bit weird to me :D

Comment (Shou) [reply lib/gui/app/modules/image-writer.js]: Done

**Comment by Shou** [comment]
Gonna rebase this on top of https://github.com/resin-io/etcher/pull/2301 and https://github.com/resin-io/etcher/pull/2295

**Comment by Shou** [comment]
Here we go

**Pull request merged**